src/Menubar.ts

```diff
@@ -281,11 +298,55 @@ export class Menubar extends EventEmitter {
         return;
       }
 
+      /*
+        Reset the alwaysOnTop value to the users value. On Windows, it's forced
+        to true while the window is visible
+      */
+      if(this._alwaysOnTopResetValue !== undefined) {
+        const isAlwaysOnTop = this._alwaysOnTopResetValue;
+        /*
+          This has to be unset before we set alwaysOnTop. Otherwise, we'll
+          handle the event as if the user changed it and force it back to true.
+        */
+        this._alwaysOnTopResetValue = undefined;
+        this._browserWindow.setAlwaysOnTop(isAlwaysOnTop);
+      }
+
       this._browserWindow.isAlwaysOnTop()
         ? this.emit('focus-lost')
         : this.hideWindow();
     });
 
+    /*
+      On Windows, we force alwaysOnTop to true when the window is visible. If
+      the user changes this while the window is visible, we can change it back to
+      true (and track that new user alwaysOnTop preference).
+    */
+    if(process.platform === "win32") {
+      let isNextAlwaysOnTopChangeEventFromUs = false;
+      this._browserWindow.on('always-on-top-changed', (event, isAlwaysOnTop) => {
+        /*
+          We no longer want to track the user's value for alwaysOnTop. This
+          happens during the blur flow, right after we reset alwaysOnTop. The
+          window may still be visible (if their preference was alwaysOnTop=true)
+        */
+        if(this._alwaysOnTopResetValue === undefined) {
+          return;
+        }
+        // Ignore if it was caused by us
+        if(isNextAlwaysOnTopChangeEventFromUs) {
+          isNextAlwaysOnTopChangeEventFromUs = false;
+          return;
+        }
+        if(this._browserWindow && this._browserWindow.isVisible()) {
+          // Otherwise, track the new value and force back to true
+          isNextAlwaysOnTopChangeEventFromUs = true;
+          this._browserWindow.setAlwaysOnTop(true);
```

Comment:
I noticed something just now. If I click the tray icon (in the tray area) but then hover over the tray area below the window, when the user changes `alwaysOnTop` to `false`, the menubar window goes behind the tray area and stays that way.

Changing it to this solves that:

```javascript
          this._browserWindow.setAlwaysOnTop(true); 
          setTimeout(() => {
            if(this._alwaysOnTopResetValue !== undefined && this._browserWindow) {
              this._browserWindow.setAlwaysOnTop(true);
            }
          });
```

`process.nextTick` works but the tray area flashes in front and then behind again very quickly. This doesn't happen with `setTimeout`.